docs/webhooks.md

```diff
@@ -0,0 +1,78 @@
+Webhooks
+========
+
+A webhook object enables you to attach events triggers to Box files and folders. These
+event triggers monitor events on Box objects and notify your application, via HTTP
+requests to a URL of your choosing, when they occur.
+
+* [Create Webhook on File](#create-a-file-webhook)
+* [Create Webhook on Folder](#create-a-folder-webhook)
+* [Get a Webhooks Information](#get-a-webhooks-information)
+* [Get all Webhooks Information](#get-all-webhooks-information)
+* [Update a Webhook](#update-a-webhook)
+* [Delete a Webhook](#delete-a-webhook)
+
+Create a Webhook on a Box File
+------------------------------
+
+Calling
+[`webhooks.create(fileID, objectType, notificationURL, triggerTypes, callback)`](http://opensource.box.com/box-node-sdk/Webhooks.html#create)
+on a file attaches an event trigger with a URL to send notifications to
+
+```js
+client.webhooks.create('759371', FILE, 'https://www.YOURWEBSITE.com', [webhookTriggerType.FILE.UPLOADED, webhookTriggerType.FILE.DOWNLOADED], callback)
```

Comment:
an enum in box-client.js I changed it to targetType.FILE and targetType.FOLDER, is that okay?